src/xrpld/rpc/handlers/Simulate.cpp

```diff
@@ -342,6 +342,15 @@ doSimulate(RPC::JsonContext& context)
         return jvResult;
     }
 
+    if (stTx->getTxnType() == ttBATCH)
+    {
+        Json::Value jvResult = Json::objectValue;
+        jvResult[jss::error] = "invalidTransaction";
+        jvResult[jss::error_exception] =
+            "Batch transactions are not supported.";
+        return jvResult;
+    }
```

Comment:
I would return `rpcNOT_IMPL` here
```suggestion
    {
        return RPC::make_error(rpcNOT_IMPL);
    }
```